src/components/Tag.jsx

```diff
@@ -0,0 +1,11 @@
+import icX from "../assets/ic_X.svg";
+import styles from "./Tag.module.css";
+
+function Tag({ children }) {
+    return (<div className={styles.tag} >
+        {children}
+        <img src={icX} />
+    </div>);
+}
```

Comment:
❗️ 수정요청
추가된 태그를 삭제할 수 있어야합니다~

```suggestion
function Tag({ children, onClick }) {
    return (
      <div className={styles.tag} >
        {children}
        <button type="button" onClick={onClick}><img src={icX} /></button>
      </div>
    );
}
```